app/code/Magento/GiftMessageGraphQl/composer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "magento/module-gift-message-graph-ql",
+  "description": "N/A",
+  "type": "magento2-module",
+  "require": {
+    "php": "~7.1.3||~7.2.0||~7.3.0",
```

Comment:
Can you update php requirements to match recently updated versions for 2.4

Reply:
**Done**